common/meta/raft-store/src/state_machine/sm.rs

```diff
@@ -248,9 +250,15 @@ impl StateMachine {
         let log_id = &entry.log_id;
 
         let sm_meta = self.sm_meta();
-        sm_meta
-            .insert(&LastApplied, &StateMachineMetaValue::LogId(*log_id))
-            .await?;
+        sm_meta.txn(move |t| {
+            t.insert(&LastApplied, &StateMachineMetaValue::LogId(*log_id))
+                .await?;
+            t.txn_tree.flush();
+            Ok(())
+        });
```

Comment:
A transaction may include more than one KeySpace, thus here it should be something like:
```rust
let txn_tree = TxnTree::new(self.tree);
txn_tree.txn(|t| {
    t.sm_meta().insert(k,v)
})
```